Cargo.toml

```diff
@@ -17,15 +17,13 @@ smallvec = "1.11.2"
 rustc-hex = "2.1.0"
 env_logger = "0.10.1"
 lru = "0.12.1"
-parking_lot = "0.12.1"
-
-bitvec = { version = "1", default-features = false }
+bitvec = { version = "1" }
```

Comment:
If you want your code to compile in no_std (require for wasm). You will need to import all of those creates with `default-features = false`, and the create a rust feature `std`. You can see us doing that in all of our `primitives` crate in madara.